VndbSharp/Structs/Models/DatabaseStats/DatabaseStats.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace VndbSharp.Structs.Models.DatabaseStats
+{
+    [JsonObject]
```

Comment:
I'm not sure this does what you expect it to... [JsonObjectAttribute](http://www.newtonsoft.com/json/help/html/JsonObjectAttributeOverrideIEnumerable.htm)